lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -77,6 +77,17 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     notImplemented("HTMLFormElement.prototype.submit", this._ownerDocument._defaultView);
   }
 
+  _doReset() {
+    if (!isConnected(this)) {
+      return;
+    }
+    const ev = this._ownerDocument.createEvent("HTMLEvents");
+    ev.initEvent("reset", true, true);
```

Comment:
`createEvent`/`initEvent` are older methods for creating events which we don't use in newer code. [This approach](https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/nodes/HTMLInputElement-impl.js#L222) is preferred instead.

Reply:
I updated these to use the current method of creating/initializing events. I hope I am not overstepping, but I took the initiative to move the dispatch of both submit/reset events into the reset/submit methods so that `html/semantics/forms/resetting-a-form/reset-form-event-realm.html` tests could also be re-enabled.